tweener/tweener.lua

```diff
@@ -12,15 +12,14 @@ local M = {}
 local TYPE_TABLE = "table"
 local TYPE_USERDATA = "userdata"
 
-
 ---Starts a tweening operation.
 ---@param easing_function easing_function
 ---@param from number
 ---@param to number
 ---@param time number
 ---@param callback fun(value: number, is_end: boolean)
 ---@param update_delta_time number|nil @Default is 1/60, the time between updates
----@return hash timer_id The created timer id, you can cancel a tween by calling timer.cancel(timer_id)
+---@return table timer_id The created timer information, you can cancel a tween by calling tweener.cancel(timer_id)
```

Comment:
Now the variable is not `timer_id` but kind of `tween_state` or kind of.

So it will read much easier, if we will have renamed it everywhere in file to match it with the code logic